### PR TITLE
Serverless pattern eventbridge pipe sqs stepfunctions filter transform cdk python

### PR DESCRIPTION
**Pull request by manikandanks2000**
*Issue #, if available:*

*Description of changes:*
This pattern will use Amazon EventBridge Pipe to accept source messages from Amazon Simple Queue Service and will apply filter and transformation before sending the target message to AWS Step Functions.
This example project implemented with CDK/Python.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by pputhran** [comment]
Can you please check the errors and re-submit please?

**Comment by manikandanks2000** [comment]
Fixed the errors on example pattern file and linkedin.  Let me know if you have any questions. Thanks

**Review by manikandanks2000 — COMMENTED** [review]
fixed the errors and resubmitted for git code review workflow for schema validation check.

File: cdk-sqs-eventbridge-pipes-stepfunctions--filter-transform-python/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      "SQS Messages will be filtered and transformed before sending to AWS Step Functions."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/cdk-sqs-eventbridge-pipes-stepfunctions--filter-transform-python",
+      "templateURL": "serverless-patterns/cdk-sqs-eventbridge-pipes-stepfunctions-filter-transform-python",
+      "projectFolder": "cdk_eventbridge_pipes_sqs_to_stepfunctions_python",
+      "templateFile": "cdk_eventbridge_pipes_sqs_to_stepfunctions_python_stack.py"
+    }
```
Comment (pputhran) [inline cdk-sqs-eventbridge-pipes-stepfunctions--filter-transform-python/example-pattern.json:21]: Can you please take a look at the `projectFolder` and `templateFile` references? They seem to be incorrect.

**Review by pputhran — CHANGES_REQUESTED** [review]

**Comment by pputhran** [comment]
@manikandanks2000 Did you get a chance to review the requested changes?

**Comment by manikandanks2000** [comment]
@pputhran  thanks for the input ..checking the changes, will confirm once complete soon.

**Comment by manikandanks2000** [comment]
@pputhran  Let me know if the changes are good to move forward. thanks

**Pull request closed without merging**